Tests/BackendIntegrationTests/OtherIntegrationTests.swift

```diff
@@ -173,7 +175,7 @@ class OtherIntegrationTests: BaseBackendIntegrationTests {
         try AvailabilityChecks.iOS15APIAvailableOrSkipTest()
 
         let result = try await self.purchases.productEntitlementMapping().entitlementsByProduct
-        expect(result).to(haveCount(18))
+        expect(result).to(haveCount(21))
```

Comment:
I'm not sure when (the audit logs aren't loading for me), but we now have 21 iOS products in the StoreKit Integration Tests app instead of 18

Reply:
I'm looking into this a little more to confirm why this has changed

Reply:
The most recently added product to the project was on June 25, so this isn't an issue with a new product being present. Continuing to dig 👀

Reply:
The test is currently failing in main with the value `18`, but passed during yesterday's release pipeline 🧐


https://app.circleci.com/pipelines/github/RevenueCat/purchases-ios/20914/workflows/68cd311d-9aec-47a8-8f55-889bef8bdabc

Reply:
@RevenueCat/coresdk thoughts?

Reply:
Josh and I took a look at some network logs and found that the backend was returning 18 mappings on Tuesday when the test passed, but is now returning 21 in the network calls to `/v1/product_entitlement_mapping`. This is happening on this branch, main, and the 4.32.2 release. We now believe that 21 is the correct number and are following up with the backend team to investigate what caused the change to take so long to appear.